src/pages/LoginForm/LoginForm.tsx

```diff
@@ -42,75 +46,86 @@ const LoginForm = () => {
-        <main className="flex w-full min-h-[560px] h-[80vh] relative">
-          <img
-            className="absolute object-cover w-full h-full top-1/2 left-1/2 translate-x-[-50%] translate-y-[-50%]"
-            src={bgLogin}
-            alt=""
-          />
-          <section className="relative z-10 flex flex-col items-center justify-center w-auto m-auto rounded-md p-7 login-form customGlass">
-            <h2 className="mb-4 text-3xl font-bold text-center text-white font-tertiaryFont">
-              Logowanie
-            </h2>
-            <Formik
-              initialValues={initialValues}
-              validationSchema={validationSchema}
-              onSubmit={handleSubmit}
-            >
-              {() => (
-                <Form className="flex flex-col w-full max-w-sm">
-                  <div className="mb-5">
-                    <InputField
-                      type="text"
-                      id="username"
-                      name="username"
-                      placeholder="E-mail"
-                    />
-                    <InputError name="username" />
-                  </div>
-                  <div className="mb-5">
-                    <InputField
-                      type="password"
-                      id="password"
-                      name="password"
-                      placeholder="Hasło"
-                    />
-                    <InputError name="password" />
-                  </div>
-                  <CheckField
-                    id="notRobot"
-                    name="notRobot"
-                    label="Nie jestem robotem"
-                  />
-                  <InputError name="notRobot" />
-                  <button
-                    type="submit"
-                    className="px-4 py-2 mt-4 font-semibold text-white rounded-lg font-secondaryFont bg-thirdColor hover:bg-fourthColor focus:outline-none focus:ring-2 focus:bg-fourthColor"
-                  >
-                    Zaloguj się
-                  </button>
-                  <div className="px-4 py-2 mt-4 text-white font-secondaryFont text-color">
-                    Nie masz konta?
-                    <Link
-                      className="font-semibold font-secondaryFont text-primaryColor"
-                      to="/registration"
-                    >
-                      {' '}
-                      Zarejestruj się
-                    </Link>
-                  </div>
-                </Form>
-              )}
-            </Formik>
-          </section>
-        </main>
-        <InfoPage />
+        {loading && <Preloader />}
+        {!loading && (
+          <>
+            <main className="flex w-full min-h-[560px] h-[80vh] relative">
+              <img
+                className="absolute object-cover w-full h-full top-1/2 left-1/2 translate-x-[-50%] translate-y-[-50%]"
+                src={bgLogin}
+                alt="Background"
+              />
```

Comment:
To moglibyśmy wydzielić do osobnego komponentu dziecka dla LoginForm, np LoginFormBackground